upgrading/topics/keycloak/changes-19_0_2.adoc

```diff
@@ -11,4 +11,43 @@ bin/kc.[sh|bat] --spi-login-protocol-openid-connect-suppress-logout-confirmation
 
 With this configuration, you can still use the logout endpoint without a user prompt.
 
-WARNING: The backwards compatibility switch will be removed in some future version - probably Keycloak 23. You are encouraged to update your clients as soon as possible as described above rather than rely on this switch.
\ No newline at end of file
+WARNING: The backwards compatibility switch will be removed in some future version - probably Keycloak 23. You are encouraged to update your clients as soon as possible as described above rather than rely on this switch.
+
+= UserInfo Endpoint Changes
+
+Error response changes::
+  The UserInfo endpoint is now returning error responses fully compliant with https://datatracker.ietf.org/doc/html/rfc6750[RFC 6750] (The OAuth 2.0 Authorization Framework: Bearer Token Usage). Error code and description (if available) are provided as `WWW-Authenticate` challenge attributes rather than JSON object fields.
+  The responses will be the following, depending on the error condition:
+  * In case no access token is provided:
++
+----
+401 Unauthorized
+WWW-Authenticate: Bearer realm="myrealm"
+----
+  * In case several methods are used simultaneously to provide access token (e.g. Authorization header + POST access_token parameter), or POST parameters are duplicated:
++
+----
+400 Bad Request
+WWW-Authenticate: Bearer realm="myrealm", error="invalid_request", error_description="..."
+----
+  * In case access token is missing `openid` scope:
```

Comment:
"an access token" might be good instead of "access token".

Reply:
done